week-1/exercises/count.js

```diff
@@ -4,3 +4,4 @@ count = count + 1;
 
 // Line 1 is a variable declaration, creating the count variable with an initial value of 0
 // Describe what line 3 is doing, in particular focus on what = is doing
+// line 3 is reassigning the value for the variable and "=" is the assignment operator
```

Comment:
Excellent vocabulary here with 'assignment operator' Areeb!

Reply:
Thank you